system/lib/pthread/pthread_create.c

```diff
@@ -155,6 +155,8 @@ void _emscripten_thread_exit(void* result) {
 
   // Not hosting a pthread anymore in this worker set __pthread_self to NULL
   _emscripten_thread_init(0, 0, 0);
+  self->tid = 0;
+  self->dead = 1;
```

Comment:
grepping `system/` I can't find any `.dead` or `->dead`. How is it used?

Reply:
Ah, looks like you removed this line meanwhile as I was reviewing...

Reply:
Sorry, this was from a previous version of the PR..  dead was removing the v1.2.2 update that just landed and then I rebased.